ts/src/bitfinex2.ts

```diff
@@ -959,7 +959,7 @@ export default class bitfinex2 extends Exchange {
         return this.parseTransfer (response, currency);
     }
 
-    parseTransfer (transfer, currency: Currency = undefined) {
+    parseTransfer (transfer: Dict, currency: Currency = undefined): TransferEntry {
```

Comment:
@samgermain again, we have to do the wrapper thing here

Reply:
Let me fix my own mistakes, please don't fix them for me